app/enterprise/changelog.md

```diff
@@ -4,6 +4,195 @@ no_search: true
 no_version: true
 ---
 
+## 2.6.0.0
+**Release date:** 2021/09/28
+
+### Features
+
+#### Enterprise
+
+#### Core
+This release includes the addition of a new schema entity validator: `mutually_exclusive`. Before, the
+`only_one_of` validator required at least one of the fields included be configured. This new entity validator allows
+only one or neither of the fields be configured.
+[#7765](https://github.com/Kong/kong/pull/7765)
+
+#### Performance
+
+For this release we've focused on performance improvement.
+
+There's a new performance workflow which periodically checks new code additions against some
+typical scenarios. [#7030](https://github.com/Kong/kong/pull/7030) [#7547](https://github.com/Kong/kong/pull/7547)
+
+Additionally, the following changes were specifically included to improve performance:
+
+- Reduced unnecessary reads of `ngx.var`.
+  [#7840](https://github.com/Kong/kong/pull/7840)
+- Loaded more indexed variables.
+  [#7849](https://github.com/Kong/kong/pull/7849)
+- Optimized table creation in Balancer.
+  [#7852](https://github.com/Kong/kong/pull/7852)
+- Reduced calls to `ngx.update_time`.
+  [#7853](https://github.com/Kong/kong/pull/7853)
+- Use read-only replica for PostgreSQL meta-schema reading.
+  [#7454](https://github.com/Kong/kong/pull/7454)
+- URL escaping detects cases when it's not needed and early-exists.
+  [#7742](https://github.com/Kong/kong/pull/7742)
+- Accelerated variable loading via indexes.
+  [#7818](https://github.com/Kong/kong/pull/7818)
+
+#### Configuration
+- Enable IPV6 on `dns_order` as unsupported experimental feature.
+  [#7819](https://github.com/Kong/kong/pull/7819).
+- The template renderer can now use `os.getenv`.
+  [#6872](https://github.com/Kong/kong/pull/6872).
+
+#### Hybrid Mode
+- Data plane is able to eliminate some unknown fields when Control Plane is using a more modern version.
+  [#7827](https://github.com/Kong/kong/pull/7827).
+
+#### Admin API
+- Add support for HEAD requests.
+  [#7796](https://github.com/Kong/kong/pull/7796)
+- Improve support for OPTIONS requests.
+  [#7830](https://github.com/Kong/kong/pull/7830)
+
+#### Plugins
+- **New plugin:** [jq](/hub/kong-inc/jq) (`jq`)
+  The jq plugin enables arbitrary jq transformations on JSON objects included in API requests or responses.
+- [Kafka Log](/hub/kong-inc/kafka-log) (`kafka-log`)
+  The Kafka Log plugin now supports TLS, mTLS, and SASL auth.
+- [Kafka Upstream](/hub/kong-inc/kafka-upstream) (`kafka-upstream`)
+  The Kafka Upstream plugin now supports TLS, mTLS, and SASL auth.
+- [Rate Limiting Advanced](/hub/kong-inc/rate-limiting-advanced) (`rate-limiting-advanced`)
+  - The Rate Limiting plugin now has a new identifier type: `path`, which allows rate limiting by matching request paths.
+  - The plugin now has a “local” strategy in the schema. The local strategy automatically sets `config.sync_rate` to -1.
```

Comment:
Local should be in backticks as in `local`

Reply:
k, will add